src/libraries/System.Net.Connections/ref/System.Net.Connections.cs

```diff
@@ -79,3 +79,24 @@ public partial class SocketsConnectionFactory : System.Net.Connections.Connectio
         protected virtual System.Net.Sockets.Socket CreateSocket(System.Net.Sockets.AddressFamily addressFamily, System.Net.Sockets.SocketType socketType, System.Net.Sockets.ProtocolType protocolType, System.Net.EndPoint? endPoint, System.Net.Connections.IConnectionProperties? options) { throw null; }
     }
 }
+namespace System.Net
```

Comment:
How does it work if we remove `System.Net.Connections` from shared framework but still depend on it? Or is it still present? Do we still expose it to the user?

Reply:
It's not exposed to the user. It's still part of the tree, but not dropped.